assets/lib/share-link/share-link.js

```diff
@@ -16,6 +16,17 @@
 			$element.on( 'click', function() {
 				openShareLink( networkName );
 			} );
+
+			// Add "Enter" and "Space" event only if the element has role=button attribute.
+			if ( 'button' === $element.attr( 'role' ) ) {
```

Comment:
I think the if statement is redundant because the role="button" is always exist.

Reply:
Currently the elements that using the library are divs, divs don't have role.

Reply:
So how will this ever work? @ManorHazaz